File: java-gradle/files/src/gatling/java/{{ .PackagePath }}/{{ .NameWord }}/cases/HttpActions.java
package {{ .Package }}.{{ .NameWord }}.cases;

import io.gatling.javaapi.core.ChainBuilder;

import static io.gatling.javaapi.core.CoreDsl.exec;
import static io.gatling.javaapi.http.HttpDsl.http;
import static io.gatling.javaapi.http.HttpDsl.status;

public final class HttpActions {

    public static final ChainBuilder getMainPage =
            exec(http("GET /").get("/").check(status().is(200)));

    private HttpActions() {
    }
}

File: kotlin-maven/files/src/test/kotlin/{{ .PackagePath }}/{{ .NameWord }}/cases/HttpActions.kt
package {{ .Package }}.{{ .NameWord }}.cases

import io.gatling.javaapi.core.ChainBuilder
import io.gatling.javaapi.core.CoreDsl.exec
import io.gatling.javaapi.http.HttpDsl.http
import io.gatling.javaapi.http.HttpDsl.status

object HttpActions {
    val getMainPage: ChainBuilder = exec(
        http("GET /").get("/").check(status().`is`(200))
    )
}
